src/components/pages/order/Navbar/Navbar.js

```diff
@@ -1,13 +1,14 @@
-import styled from 'styled-components';
-import { theme } from '../../../../theme';
-import Logo from '../../../reusable-ui/Logo';
-import Profile from './Profile';
+import styled from "styled-components";
+import { theme } from "../../../../theme";
+import Logo from "../../../reusable-ui/Logo";
+import NavbarRightSide from "./NavbarRightSide";
+import { refreshPage } from "../../../../utils/window";
 
-export default function Navbar({ name }) {
+export default function Navbar({ username }) {
   return (
     <NavbarStyled>
-      <Logo />
-      <Profile name={ name } />
+      <Logo className="logo-order-page" onClick={() => refreshPage()} />
```

Comment:
```suggestion
      <Logo className="logo-order-page" onClick={refreshPage} />
```

pas besoin de fonction fléchée inline car là refreshPage ne prend pas de paramètre